### ISSUE TITLE
Your VS Code installation appears to be corrupt. Please reinstall. 

### ISSUE THREAD

**Issue by kris-araptus** [robb0wen/synthwave-vscode#314]
This does not work!!!! AND there's no open issues titled with what everyone has been experiencing since the plugin came out. 

(To remove corruption warning and [unsupported] from title-bar

Because enabling the glow modifies core files, VS code will interpret this as the core being 'corrupted' and you may see an error message on restarting your editor. You can safely dismiss this message, or remove it entirely with the [Fix VSCode Checksums](https://marketplace.visualstudio.com/items?itemName=lehni.vscode-fix-checksums) extension.

Upon installation of 'Fix VSCode Checksums', open the command palette and execute Fix Checksums: Apply. You will need to completely restart VSCode after execution, reopening without fully exiting might not be enough.
Updates)

I've tried this plugin a few times over there years and got it to work but the annoyance of having to see that notification got to me. 
For how old this plugin is this issue should have been addressed by now. 

reinstalling!! Thanks!!!

**Comment by Derjyn** [comment]
Yup, can second this. Did multiple cleanups and reinstalls, and once Synthwave is installed, it corrupts the install. Please get on top of this - I'd hate to find this cool theme is malicious.

**Comment by stjema** [comment]
same.

**Comment by cmedinadeveloper** [comment]
Same here, please fix this. I uninstalled the theme and followed the steps in the readme and the error is still showing.

**Comment by stjema** [comment]
guys its normal this message is normal. just hide that notification.
everybody gets this message.

**Comment by Derjyn** [comment]
> guys its normal this message is normal. just hide that notification. everybody gets this message.

No, no it's not normal. Having to ignore a message? Okay, not a big deal. But it's not normal or elegant. Plenty of other themes with advanced bells and whistles follow standards and run without issue.

The ground-truth here, is that the theme and fanciness need to get some love and updates.

**Comment by stjema** [comment]
glow text. impossible to not get warning of broken code.
u cant add glowing to vsc without that warning in vsc.

**Comment by Derjyn** [comment]
> u cant add glowing to vsc without that warning in vsc.

Correct, and that stems from the same issues that arise with use of the handy extension:

https://github.com/be5invis/vscode-custom-css

This is known, and there have been multiple discussions about adding support for more CSS in VS Code, negating the need for such a _(hacky)_ extension that can cause some headaches. Again, this solution is not elegant and hopefully we'll get support for more CSS awesomeness in the future.

All that being said, having the editor __crash__ with Synthwave going shouldn't be happening, nor should the corruption message. This can be fixed. Other themes have updated and fixed a similar issue. This isn't about that common warning message that power users are familiar with.

**Comment by Derjyn** [comment]
As a follow up, it's possible to use another extension, and CSS to achieve a glow effect. As of writing, no crashes, no warnings.

https://github.com/drcika/apc-extension

```
"apc.stylesheet": {
    "body": {
        "text-shadow": "0 0 3px red"
    }
}
```

![image](https://github.com/user-attachments/assets/14a1236e-1b99-4ea9-b216-44c2b1fe9556)

So again, Synthwave needs some love and updates to get on deck and kick down those cool effects without all the pain.

**Comment by goosewobbler** [comment]
Thanks for the tip @Derjyn, I started using APC.  You can get the glow working in APC by using a similar approach to the old version of Synthwave '84 with the [vscode-custom-css](https://github.com/be5invis/vscode-custom-css) extension.

It's also interesting to see what the Synthwave glow looks like when used with other themes, haha.

@robb0wen What do you think to the idea of reverting Synthwave to CSS-only and recommend users who want the glow to use APC?  I can put together a PR if it helps.
